src/mame/misc/changyu.cpp

```diff
@@ -132,11 +134,16 @@ void changyu_state::videoram_w(offs_t offset, u8 data)
 
 void changyu_state::video_start()
 {
-	m_bg_tilemap = &machine().tilemap().create(*m_gfxdecode, tilemap_get_info_delegate(*this, FUNC(changyu_state::get_bg_tile_info)), TILEMAP_SCAN_ROWS, 8, 8, 64, 32);
+	m_bg_tilemap = &machine().tilemap().create(*m_gfxdecode, tilemap_get_info_delegate(*this, FUNC(changyu_state::get_bg_tile_info)), TILEMAP_SCAN_ROWS, 8, 8, 72, 32);
 
 //  m_bg_tilemap->set_transparent_pen(0);
 }
 
+VIDEO_START_MEMBER(changyu_state, changyu2)
+{
+	m_bg_tilemap = &machine().tilemap().create(*m_gfxdecode, tilemap_get_info_delegate(*this, FUNC(changyu_state::get_bg_tile_info)), TILEMAP_SCAN_ROWS, 8, 8, 64, 32);
+}
+
```

Comment:
This intermix between `video_start` functions is quite ugly to see, I believe `VIDEO_START_MEMBER` is also de-facto deprecated. A quick way to fix this inconvenient is making `changyu2` a subclassed state, overriding `video_start` there. Or, just declare these tilemaps in the `machine_config`s instead, as per https://github.com/mamedev/mame/blob/09c80cf7943687ea885e3574df940f398297005d/src/mame/atari/atarig42.cpp#L571-L573

Some extra considerations:
1. `changyu` most likely sets 72x28 in the 6845 functions, 0x800 bytes / 72 = 28. 72x32 would overrun the videoram.
2. tilemap use here is just for debugging, it should really use 6845 display callback functions instead.